meta-balena-common/recipes-containers/resin-supervisor/resin-supervisor/update-resin-supervisor

```diff
@@ -69,7 +76,9 @@ error_handler() {
 
     # If docker pull fails, start the old supervisor again and exit
     rm -rf $UPDATECONF
-    systemctl start resin-supervisor
+    if [[ "${START_STOP_SUPERVISOR}" -eq 1 ]]; then
```

Comment:
I would prefer if the test conditions use the POSIX compliant single brackets instead of the double brackets. The reason is that BalenaOS is moving towards minimizing the core root filesystem externalizing functionality into system apps, host extensions etc. Ideally at some point it could get rid of bash and using busybox or another small embedded shell, and having POSIX compliant scripts will make that easier.
Functionally, it makes no difference using one or the other.

Reply:
Updated, thanks for the note!